src/games/strategy/engine/framework/startup/ui/PBEMSetupPanel.java

```diff
@@ -204,6 +205,7 @@ private void loadForumPosters(final GameData data) {
     forumPosters.add((IForumPoster) findCachedOrCreateNew(NullForumPoster.class));
     forumPosters.add((IForumPoster) findCachedOrCreateNew(AxisAndAlliesForumPoster.class));
     forumPosters.add((IForumPoster) findCachedOrCreateNew(TripleAWarClubForumPoster.class));
```

Comment:
I'd remove this in another PR...
I believe, that the NullForumPoster is actually the "none entry" in the combobox